_data-prepper/common-use-cases/anomaly-detection.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: default
+title: Anomaly detection
+parent: Common use cases
+nav_order: 30
+---
+
+# Anomaly detection
+
+You can use Data Prepper to train models and generate anomalies in near real-time on time-series aggregated events. You can generate anomalies either on events generated within the pipeline, or on events coming directly into the pipeline, like OpenTelemetry metrics. You can feed these tumbling window aggregated time-series events to the [`anomaly_detector` processor]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/anomaly-detector/), which trains a model and generates anomalies with a grade score. Then write the anomalies to a separate index to create document monitors and trigger fast alerting.
+
+## Metrics from logs 
+
+The following pipeline receives logs using an HTTP source like FluentBit, extracts important values from the logs by matching the value in the `log` key against the grok [Apache Common Log Format](https://httpd.apache.org/docs/2.4/logs.html#accesslog), and then forwards the grokked logs to both the `log-to-metrics-pipeline sub-pipeline` and to an OpenSearch index named `logs`.
```

Comment:
First sentence: "from" instead of "using"? Should "Grok" be capitalized?

Reply:
USe "grok" lowercase when it's not a proper name. The proper name "Grok" is X's AI model. The OpenSearch processor is named `grok`. When we use the processor name, I think we should put it in code format so that it's clear to the reader when we're referring to the OpenSearch processor. Otherwise, lowercase "grok" when it's commonly used.